PatientWebApplication/HealthClinic.DAL/Service/RegularAppointmentService.cs

```diff
@@ -145,5 +153,62 @@ public Boolean isTermNotAvailable(DoctorUser doctor, TimeSpan time, String dateT
             return false;
         }
 
+        public List<DoctorAppointment> SimpleSearchAppointments(AppointmentReportSearchDto appointmentReportSearchDto)
+        {
+            List<DoctorAppointment> appointments = GetAppointmentsForPatient(appointmentReportSearchDto.PatientId);
+
+            appointments = SearchForDate(appointments, appointmentReportSearchDto);
+
+            appointments = SearchForDoctorNameAndSurname(appointments, appointmentReportSearchDto);
+
+            appointments = SearchForAppointmentType(appointments, appointmentReportSearchDto);
+
+            return appointments;
+
+        }
+
+        private List<DoctorAppointment> SearchForDoctorNameAndSurname(List<DoctorAppointment> appointments, AppointmentReportSearchDto appointmentSearchDto)
+        {
+            if (!appointmentSearchDto.DoctorNameAndSurname.Equals(""))
```

Comment:
It is necessary to add a method that checks if it is an empty string, and replace it in the if statement.